Userland/Libraries/LibGfx/PNGWriter.cpp

```diff
@@ -123,23 +146,12 @@ void NonCompressibleBlock::update_adler(u8 data)
     m_adler_s2 = (m_adler_s2 + m_adler_s1) % 65521;
 }
 
-void PNGWriter::add_chunk(PNGChunk const& png_chunk)
+void PNGWriter::add_chunk(PNGChunk& png_chunk)
```

Comment:
You are cahngeing the data the caller might use later, is that inentional?

Reply:
Yes, PNGWriter modifies the content of the png_chunk in this function. I am not sure if there is a cleaner way to do this.